Cargo.toml

```diff
@@ -18,6 +18,13 @@ pre-release-commit-message = "cargo: coreos-installer release {{version}}"
 pre-release-hook = ["make", "docs", "clean"]
 tag-message = "coreos-installer v{{version}}"
 
+# See https://github.com/cgwalters/cargo-vendor-filterer
+[package.metadata.vendor-filter]
+# Right now the vendor filter doesn't support wildcards, so use this as a stand-in.
+# (In practice we don't have any architecture-specific crate deps, only platform-specific)
```

Comment:
Phrasing is a bit confusing here, since coreos-installer does have arch-specific deps (`mbrman` and `rand`).

Reply:
Ohh wow.  I had missed that.  This seems to actually be pretty unusual; using a crude grep, the rest of the dependency tree almost always just conditionalizes on wasm32, with one small exception for something called "hermit" that I have no idea what that is:

```
$ cd vendor
$ rg '\[target.*target_arch.*dependencies'
web-sys/Cargo.toml
2558:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-futures]
2561:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]

wasm-bindgen-futures/Cargo.toml
41:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.futures-channel-preview]
44:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.futures-lite]
48:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]

uuid/Cargo.toml
149:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-lib]
153:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]

wasm-bindgen/Cargo.toml
63:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.js-sys]
66:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.serde_derive]
69:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-futures]
72:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]
75:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test-crate-a]
78:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test-crate-b]

wasm-bindgen/guide/src/reference/rust-targets.md
53:[target.'cfg(target_arch = "wasm32")'.dependencies]

tracing/Cargo.toml
110:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]

tokio/Cargo.toml
182:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.proptest]
185:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.rand]
188:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.socket2]
191:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]

reqwest/Cargo.toml
210:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.async-compression]
216:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.cookie_crate]
221:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.cookie_store]
225:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.encoding_rs]
228:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.futures-core]
232:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.futures-util]
236:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.h2]
239:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.http-body]
242:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.hyper]
253:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.hyper-rustls]
258:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.hyper-tls]
262:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.ipnet]
265:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.lazy_static]
268:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.log]
271:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.mime]
274:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.native-tls-crate]
279:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.percent-encoding]
282:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.pin-project-lite]
285:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.proc-macro-hack]
289:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.rustls]
294:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.rustls-native-certs]
298:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.rustls-pemfile]
302:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.tokio]
310:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.tokio-native-tls]
314:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.tokio-rustls]
318:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.tokio-socks]
322:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.tokio-util]
331:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.trust-dns-resolver]
335:[target."cfg(not(target_arch = \"wasm32\"))".dependencies.webpki-roots]
339:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.brotli_crate]
343:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.doc-comment]
346:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.env_logger]
349:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.hyper]
362:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.libflate]
365:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.serde]
369:[target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.tokio]
377:[target."cfg(target_arch = \"wasm32\")".dependencies.js-sys]
380:[target."cfg(target_arch = \"wasm32\")".dependencies.serde_json]
383:[target."cfg(target_arch = \"wasm32\")".dependencies.wasm-bindgen]
386:[target."cfg(target_arch = \"wasm32\")".dependencies.wasm-bindgen-futures]
389:[target."cfg(target_arch = \"wasm32\")".dependencies.web-sys]
406:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen]
410:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]

num_cpus/Cargo.toml
23:[target."cfg(all(any(target_arch = \"x86_64\", target_arch = \"aarch64\"), target_os = \"hermit\"))".dependencies.hermit-abi]

js-sys/Cargo.toml
35:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-futures]
38:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]
41:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.web-sys]

getrandom/Cargo.toml
66:[target."cfg(all(target_arch = \"wasm32\", target_os = \"unknown\"))".dependencies.js-sys]
70:[target."cfg(all(target_arch = \"wasm32\", target_os = \"unknown\"))".dependencies.wasm-bindgen]
75:[target."cfg(all(target_arch = \"wasm32\", target_os = \"unknown\"))".dev-dependencies.wasm-bindgen-test]

fastrand/Cargo.toml
32:[target."cfg(target_arch = \"wasm32\")".dependencies.instant]
34:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.getrandom]
38:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.instant]
42:[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]

flate2/Cargo.toml
92:[target."cfg(all(target_arch = \"wasm32\", not(target_os = \"emscripten\")))".dependencies.miniz_oxide]
$ 
```